master/reference/advanced/etcd-rbac/calico-etcdv2-paths.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Calico key and path prefixes in etcd v2
+title: Calico key and path prefixes in Etcd v2
```

Comment:
@bcreane actually, this is not what the style guide recommends :D....etcd is always lowercased (per the vendor)

Reply:
nuts.